src/routes/sessions.js

```diff
@@ -22,8 +22,20 @@ module.exports = function (app, opts, dependencies) {
     return response.status(401).send(body);
   }
 
-  async function processLogin({ useGoogleAuthentication, renderingId, authData, response }) {
+  async function processLogin({
+    useGoogleAuthentication,
+    renderingId,
+    authData,
+    response,
+    isRegistration,
+    projectId,
+    twoFactorToken,
+  }) {
     try {
+      if (isRegistration && !twoFactorToken) {
+        throw new Error();
+      }
```

Comment:
In the front the "Enable" does not seem to be disabled so it looks like I can submit an empty string token. 
Maybe the button should be disable without the 6 digits entered.
But an explicit error message would be even greater.

Reply:
https://github.com/ForestAdmin/forestadmin/pull/944/commits/1e517a3f91b83087da7627c53207a270b6d9fdb2